src/documents/css/idg.css.styl

```diff
@@ -434,9 +434,78 @@ setActiveCategory(inputColor) {
     list-style-type: none;
 }
 
+/* Footer logos */
+
+footer svg {
+    width: 6rem;
+    height: 7rem;
+    fill: #fff;
+    stroke: header-footer-background;   
+}   
+
+/* remove hover and focus from the anchors (override template), apply to svgs only */
+a:hover.footer-logo-links { 
+    background: none; 
+}
+
+footer a:hover svg {
```

Comment:
You can change this to "a:hover.footer-logo-links svg" since it's the same as "footer a:hover svg". This way it's consistent with how we address the other styles.